tests/Fixer/Whitespace/StatementIndentationFixerTest.php

```diff
@@ -1072,6 +1072,61 @@ public function foo() {
 }',
             ['stick_comment_to_next_continuous_control_statement' => true],
         ];
+
+        yield 'if/else/elseif without braces' => [
```

Comment:
can we add 
```php
if (1)
if (2)
echo "a";
else
echo "b";
```
?


I would love to see if it's fixing to proper variant,
```php
if (1)
    if (2)
        echo "a";
    else
        echo "b";
```
or
```php
if (1)
    if (2)
        echo "a";
else
    echo "b";
```

Reply:
Nice catch! This is currently fixed as the second snippet while it should be fixed as the first one. I think it should be fairly easy to fix but I'm going to wait until we decide [whether we want to merge this PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7663#discussion_r1439771856) at all.

Reply:
Fixed